src/keydown_content.ts

```diff
@@ -1,119 +0,0 @@
-/** Shim for the keyboard API because it won't hit in FF57. */
-
-import * as Messaging from "./messaging"
-import * as msgsafe from "./msgsafe"
-import { isTextEditable, getAllDocumentFrames } from "./dom"
-import { isSimpleKey } from "./keyseq"
-
-function keyeventHandler(ke: KeyboardEvent) {
-    // Ignore JS-generated events for security reasons.
-    if (!ke.isTrusted) return
```

Comment:
The removal of this line is what led to https://github.com/tridactyl/tridactyl/security/advisories/GHSA-7qr7-93pf-hr8f. Mentioning https://github.com/tridactyl/tridactyl/issues/1833 here so we get a link back to this line.